scripts/build.py

```diff
@@ -334,17 +419,30 @@ def build_minified_third_party_libs():
 
     print 'Building minified third party libs...'
 
-    third_party_js = os.path.join(
+    STAGING_THIRD_PARTY_JS = os.path.join(
```

Comment:
STAGING_THIRD_PARTY_JS_FILEPATH, STAGING_THIRD_PARTY_CSS_FILEPATH

(Names of things should end with nouns.)

Reply:
Done. Thanks!